modules/recon/domains-credentials/scylla.py

```diff
@@ -19,9 +19,8 @@ def module_run(self, domains):
         base_url = 'https://scylla.sh/search'
         headers = {'Accept': 'application/json'}
         size = self.options['size']
-        for domain in domains:
-            page = 0
-            _from = size*page
+        start = 1
+        for email in emails:
```

Comment:
I'm pretty sure this can't work as is. The module uses `domain` not `email` so this should break.

Reply:
Ya got a little too fast on the trigger with the commit. Should be fixed now and fully tested.